### PR TITLE
[#12314] Popover appears in front of modal blocking buttons

### LINKED ISSUE #12314: Popover appears in front of modal blocking buttons
- **Environment**: 

V8.25.0

- **Issue**

telegram-cloud-photo-size-5-6138667426573826030-y

### PR DESCRIPTION
**Pull request by Mex7180**
Fixes #12314

**Outline of Solution**

 
- add mouse enter event to enable the tooltip
- close tooltip when the button to save the responses is clicked

**Laptop size:**
Bildschirmfoto 2023-09-04 um 02 13 40
Bildschirmfoto 2023-09-04 um 02 13 54

**Mobile size:**
Bildschirmfoto 2023-09-04 um 02 14 16
Bildschirmfoto 2023-09-04 um 02 14 27

### CODE REVIEW

**Comment by Mex7180** [comment]
Ready for review :D

**Comment by Mex7180** [comment]
Ready for review :D

**Review by cedricongjh — APPROVED** [review]
LGTM, thank you for your contribution to TEAMMATES!

File: src/web/app/components/question-submission-form/question-submission-form.component.html [hunk 1]
```diff
@@ -214,7 +214,8 @@ <h2 class="question-details"><b>Question {{ model.questionNumber }}: </b>{{ mode
         <div class="col-12 text-center">
           <button id="btn-submit-qn-{{ model.questionNumber }}" type="submit" class="btn btn-success"
                   ngbTooltip="You can save your responses for this question at any time and come back later to continue."
-                  (click)="saveFeedbackResponses()" [disabled]="isSavingResponses || isSubmissionDisabled">
+                  #tooltip="ngbTooltip" (mouseenter)="tooltip.open()"
+                   (click)="saveFeedbackResponses(); tooltip.close();"[disabled]="isSavingResponses || isSubmissionDisabled">
```
Comment (jasonqiu212) [inline src/web/app/components/question-submission-form/question-submission-form.component.html:218]:
```suggestion
                  (click)="saveFeedbackResponses(); tooltip.close();"[disabled]="isSavingResponses || isSubmissionDisabled">
```

**Review by jasonqiu212 — CHANGES_REQUESTED** [review]
Thanks for contributing to TEAMMATES! @Mex7180 

Just a very small nitpick on the indentation! Other than that, LGTM!

**Review by jasonqiu212 — APPROVED** [review]
LGTM! Thanks for contributing to TEAMMATES!

**Pull request merged**